### PR TITLE
Cc 478 initial terraform

### PR DESCRIPTION
**Pull request by mcolwellch**
CC-478 node-18 upgrade and Terraform config.

### CODE REVIEW

File: terraform/groups/ecs-service/variables.tf [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  type = number
+}
+
+variable "search_web_cookie_name" {
+  type = string
+}
+
+variable "search_web_version" {
+  type        = string
+  description = "The version of the certificates orders web container to run."
```
Comment (VaughanJackson123) [inline terraform/groups/ecs-service/variables.tf:84]: `search web` :)

File: terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars [hunk 2]
```diff
@@ -0,0 +1,17 @@
+
+# Account environment variables
+aws_profile = "development-eu-west-2"
+environment = "cidev"
+
```
Comment (VaughanJackson123) [inline terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars:5]: Looks like we don't need `kms_alias = "alias/development-eu-west-2/environment-services-kms"` here then?

**Review by VaughanJackson123 — APPROVED** [review]

File: terraform/groups/ecs-service/variables.tf (hunk 1 above)
Comment (rmcmurray-ch) [reply terraform/groups/ecs-service/variables.tf]: updated this now 👍

File: terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars (hunk 2 above)
Comment (VaughanJackson123) [reply terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars:5]: * Actually I see now this that @l-seviour removed this variable intentionally in [this commit](https://github.com/companieshouse/search.web.ch.gov.uk/commit/ab731c9bbbb5724b0a1e7d2e815a86979545e4bf). 
* Hence definitely no problem here. 👍

**Pull request merged**